include/picongpu/param/particle.param

```diff
@@ -46,42 +46,50 @@ namespace picongpu
          *  unit: none */
         constexpr float_X MIN_WEIGHTING = 10.0;
 
-        /** Number of maximum particles per cell during density profile evaluation.
+        /** Number of maximum macro-particles per cell during density profile evaluation.
          *
-         * Determines the weighting of a macro particle and with it, the number of
-         * particles "sampling" dynamics in phase space.
+         * Determines the weighting of a macro particle as well as the number of
+         * macro-particles which sample the evolution of the particle distribution
+         * function in phase space.
          */
         constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 2u;
```

Comment:
TYPICAL_NUMBER_PARTICLE_PER_CELL [does not influence weighting of a particles at all](https://github.com/ComputationalRadiationPhysics/picongpu/blob/f1f9a2d5781a53469a2124dbaa5872a9e8aaba49/include/picongpu/particles/InitFunctors.hpp#L58-L68).

Reply:
Ahhh, :cursing_face:. Thanks.

Reply:
don't worry, I built two separate bugs into FLYonPIC because I did not know either

Reply:
I moved this documentation text to the numParticlesPerCell variable within the startPosition functors.